bin/MadGraph5_aMCatNLO/cards/production/2017/5TeV/GammaGammaToQQ/GammaGammaTocc/GammaGammaTocc_run_card.dat

```diff
@@ -0,0 +1,295 @@
+#                                                                    *
+#  This file is used to set the parameters of the run.               *
+#                                                                    *
+#  Some notation/conventions:                                        *
+#                                                                    *
+#   Lines starting with a '# ' are info or comments                  *
+#                                                                    *
+#   mind the format:   value    = variable     ! comment             *
+#                                                                    *
+#   To display more options, you can type the command:               *
+#      update full_run_card                                          *
+#*********************************************************************
+#
+#*******************                                                 
+# Running parameters
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1	= run_tag ! name of the run 
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  50000	= nevents ! Number of unweighted events requested 
+ 1000 = iseed ! rnd seed (0=assigned automatically=default))
+#*********************************************************************
+# Collider type and energy                                           *
+# lpp: 0=No PDF, 1=proton, -1=antiproton, 2=photon from proton,      *
+#                                         3=photon from electron     *
+#*********************************************************************
+  2	= lpp1 ! beam 1 type 
+  2	= lpp2 ! beam 2 type
+  2510.0  = ebeam1 ! beam 1 total energy in GeV
+  2510.0  = ebeam2 ! beam 2 total energy in GeV
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+     0.0     = polbeam1 ! beam polarization for beam 1
+     0.0     = polbeam2 ! beam polarization for beam 2
+
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+  nn23lo1	= pdlabel ! PDF set                                     
```

Comment:
Is this intended ?

Reply:
Right, this is what PDF used in the provided LHE files from the authors (for the gg->tautau process)

Reply:
How will you use this? pLHE + GENSIM? Which tune are you going to use?

Reply:
I was thinking wmLHEGS; CP5 as in HIN-HINPbPbAutumn18pLHE-00013, etc